lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -68,6 +68,54 @@ def stub_email_alert_api_has_subscription(
           .to_return(status: 200, body: response)
       end
 
+      # Stubs the API responses as if each subscription happened in the order they are passed.
+      # Useful if you need to query the '/latest' endpoint of a subscription.
+      # Takes an array of hashes.
+      #
+      # @example
+      #  stub_email_alert_api_has_subscriptions([
+      #    {
+      #      id: 'id-of-my-subscriber-list',
+      #      frequency: 'weekly',
+      #      ended: true,
+      #    },
+      #    {
+      #      id: 'id-of-my-subscriber-list',
+      #      frequency: 'daily',
+      #    },
+      #  ])
+      #
+      # @param subscriptions [Array]
+      def stub_email_alert_api_has_subscriptions(subscriptions)
```

Comment:
As the last subscription in the list should be the "latest" I wonder whether there's any point in passing `ended` into the hash, and instead automatically set it to true on all but the last subscription in the list?

Reply:
Still worth passing it to make the tests clear, I think - but can definitely override all but the last subscription to be `ended: true`. 👍

Reply:
I now set `ended: true` by default in `apply_subscription_defaults`.